### DISCUSSION TITLE
webpack loaded the chunk resource, but report ChunkLoadError: Loading chunk [chunkname] failed.

### DISCUSSION THREAD

**Discussion by Angeliaxu** [webpack/webpack#14520] [Q&A]
# Bug report
I have a microfronted app，the base app used webpack4+vue2+vuerouter3+qiakun2，and the subapp is same as the base.
recently i upgrade the base app to webpack5，when i load the subapp，i clicked the button to get lazy load chunk, i found the chunk resource successfuly loaded。but webpack told me message as the title described。can anyone told me what’s the  reason。   

**What is the current behavior?**
![image](https://user-images.githubusercontent.com/27046577/137897608-df3a9285-eaef-465b-bfab-080a56549f6c.png)

![image](https://user-images.githubusercontent.com/27046577/137897618-63538e31-776f-4b95-9fd9-33c2a6fffdd8.png)

my subapp webpack config
```
 devServer: {
        port: process.env.DEFAULT_PORT,
        disableHostCheck: true,
        headers: {
            'Access-Control-Allow-Origin': '*', // 主应用获取子应用时跨域响应头
        },
    },
    publicPath: process.env.BASE_URL,
    css: {
        loaderOptions: {
            sass: {
                data: '@import "@/style/settings.scss";',
            },
        },
    },
    configureWebpack: {
        output: {
            library: `${name}-[name]`,
            libraryTarget: 'umd',
            jsonpFunction: `webpackJsonp_${name}`,
            globalObject: 'window',
        },

    },
```

the chunk relation
![image](https://user-images.githubusercontent.com/27046577/137898793-39d4364f-1e00-4ec2-a9e8-8328b07ccb03.png)

![image](https://user-images.githubusercontent.com/27046577/137898683-4ad2c0c1-47a2-401f-a1ff-065544991f2b.png)

![image](https://user-images.githubusercontent.com/27046577/137898914-27258cdd-254f-4a7d-8278-9e4be65c6052.png)

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Other relevant information:**
webpack version:  
Node.js version:  v14.16.0
Operating System: 
Additional tools:

**Comment by alexander-akait** [comment]
Please provide reproducible example, thanks
